### ISSUE TITLE
Don't Launch at startup

### ISSUE THREAD

**Issue by soundsnow** [Swordfish90/Lemuroid#911]
Force stop Lemuroid
Restart device
Notification Lemuroid: updating cores from play

**Comment by david79111** [comment]
yeah I agree. Updating cores should be done manually or there should be an option wether it happens automatically or manually, but apart from that it should only happen if the user opens the app.

Lemuroid always trying to update the cores out of nowhere bothers me too.
